back/src/Guide/Database/Queries/Update.hs

```diff
@@ -95,7 +95,7 @@ updateCategoryRow catId f = do
   when (old_categoryRowItemsOrder /= new_categoryRowItemsOrder) $ do
     let statement :: Statement (Uid Category, [Uid Item]) ()
         statement = [execute|UPDATE categories SET items_order = $2 WHERE uid = $1|]
-    lift $ HT.statement (catId, new_categoryRowItemsOrder) statement
+    lift $ HT.statement (catId, nub new_categoryRowItemsOrder) statement
```

Comment:
To exclude id duplicates.
It happens when adding category with full CategoryRow converted with `categoryToCategoryRow` and then adding items. In that case item order will be updated two times. 
Both `categoryToCategoryRow` and `insertItem` updates itemOrder.

Reply:
`categoryToCategoryRow` does not exist, are you talking about some other function?

Reply:
yes, `categoryToRowCategory`